src/ShapeCrawler/Texts/Field.cs

```diff
@@ -81,6 +82,11 @@ private Color ParseTextHighlight()
 
     private void UpdateTextHighlight(Color? color)
     {
+        if (color == null)
```

Comment:
I think, it doesn't look logical: the method accepts nullable parameter but checks it for `null`. If we are going to do such a check, it is probably better to do it at a higher-level public method. It keeps the internal logic clean and reduces redundancy.

Reply:
Yup. In fact, probably would be better to push non-nullable all the way up to the public interface right? Null has no useful meaning here. In the original code, sending in null will just throw a exception on the cast to (non-nulllable) `Color`

Reply:
Right, but for me, it is not yet clear what is better way in Setter of nullable _property_. Maybe something like the below code.

![image](https://github.com/ShapeCrawler/ShapeCrawler/assets/5969304/b5078c6b-202d-45eb-9f7c-549a1f44f0a9)

Feel free to make suggestions for more elegant refactoring.

Reply:
How about making `TextHighlightColor` just be non-nullable? In fact, `ParseTextHighlight()` already returns a non-nullable `Color`. Setter is going to throw an exception. Nullable `Color` seems incorrect in this case.

Reply:
Non-nullable `TextHighlightColor` also suits, but then I would suggest introducing some new `Color.NoColor`:

![image](https://github.com/ShapeCrawler/ShapeCrawler/assets/5969304/b87c7535-9c56-46f0-a12c-3edc25bc0122)

I believe it should be possible for the user to know if the text is highlighted.

Reply:
> I believe it should be possible for the user to know if the text is highlighted.

Totally agree. But that would be outside the scope of a warning-fix PR, yes?

Furthermore, from looking at the code, I believe it's possible for user to know the text is highlighted or not, because TextHighlightColor will return Color.Transparent if there is no text highlight element. There doesn't seem to be a way to REMOVE the text highlight element, although perhaps setting it to Color.Transparent will have the same effect. This seems like a good place for someone to dig in who is interested in text highlighting.

Also, I am seeing now perhaps why Color was nullable to start with. `ParagraphLineBreak` implements `IParagraphPortion`, but cannot provide `Hyperlink` or 'TextHighlightColor`. So returns null, and throws on set. I changed to throw on get as well.

Reply:
You are right: it is outside the scope of this PR. Let's let `Color.Transparent` act as “No Color” for the meantime.

There is an idea to name properties, actions, etc. as they are represented in PowerPoint. We will research this in the separate issue https://github.com/ShapeCrawler/ShapeCrawler/issues/690